docs/usage/Config.md

```diff
@@ -21,6 +21,19 @@ It can also be invoked while overriding the `baselines` parameter.
 scubagoggles gws --config basic_config.yaml -b gmail chat
 ```
 
+## Omit Policies

```

Comment:
When testing my first inclination was to use a combination of the root_config.yaml I previously created from #413 and the new omit_policies.yaml config file: `scubagoggles gws --config config-dir/root_config.yaml config-dir/omit_policies.yaml` which isn't supported. 

Perhaps in the docs we indicate something like: "Only one config file is needed. Here's the root config we provide to start, and if needed here's additional functionality like policy omission which can be added on top of the existing parameters."

Reply:
Done